js/main.js

```diff
@@ -1,115 +1,32 @@
-(function() {
+$(document).ready(function(){
     "use strict";
 
-    function loadPics() {
```

Comment:
Sorry, I may have been misleading. The parent IIFE should remain, the inner one is the one I'd like to replace with `$(document).ready()`.

```javascript
(function($){ // <- The $ here is our custom alias to the jQuery global included below.
  $(document).ready(function(){
    // app code here
  });
})(jQuery); // <- This is how we gain access of globals inside our IIFE, jQuery in this case

```